notebooks/sam_tests/cmip6_density_transform_pipeline_dev.ipynb

```diff
@@ -8,7 +8,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #109.    dask.config.set(pool=ThreadPool(32))
</pre><p>Just noting that with the new dask version this might not be necessary anymore. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/cmip6_omz/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>